member.php

```diff
@@ -1601,8 +1602,35 @@
 			$password_length = 8;
 		}
 
-		$password = random_str($password_length);
-		$logindetails = update_password($user['uid'], md5($password), $user['salt']);
+		// Set up user handler.
+		require_once 'inc/datahandlers/user.php';
+		$userhandler = new UserDataHandler('update');
+
+		$userhandler->set_data(array(
+			'uid'		=> $user['uid'],
+			'username'	=> $user['username'],
+			'email'		=> $user['email'],
+			'password'	=> random_str($password_length)

```

Comment:
I don't think those are the same lines, that is why. Otherwise GH handles it correctly or causes a conflict.

So those are the only lines I should "replicate"?
https://github.com/mybb/mybb/pull/1868/files#diff-c6480c1a3fe8c1a339ac23fcdc05d97dR132
https://github.com/mybb/mybb/pull/1868/files#diff-c6480c1a3fe8c1a339ac23fcdc05d97dR135